WEB-INF/lib/auth/Auth_ldap.class.php

```diff
@@ -137,36 +137,35 @@ function authenticate($login, $password)
         return false;
       }
 
-      if ($member_of) {
-        // get groups
+      $filter = 'samaccountname='.Auth_ldap::ldap_escape(str_replace('@' . $this->params['default_domain'], '', $login));
```

Comment:
Any AD user has the login format `<user.name>@ad.domain.tld`.  
The filter for queriing the AD use the field `samAccountName `. But this field doesn't contain the domain part. That's the reason why I removed it.